dataclients/kubernetes/clusterclient.go

```diff
@@ -358,10 +358,11 @@ func (c *clusterClient) LoadRouteGroups() ([]*definitions.RouteGroupItem, error)
 		return nil, err
 	}
 
+	routeGroupValidator := &definitions.RouteGroupValidator{}
 	rgs := make([]*definitions.RouteGroupItem, 0, len(rgl.Items))
 	for _, i := range rgl.Items {
 		// Validate RouteGroup item.
-		if err := definitions.ValidateRouteGroup(i); err != nil {
+		if err := routeGroupValidator.Validate(i); err != nil {
```

Comment:
This will break api, should I increase version?

Reply:
I think we should make routeGroupValidator a field of clusterClient

Reply:
I just bumped minor version in https://github.com/zalando/skipper/pull/2505 so may you don't have to.